routes/user.js

```diff
@@ -122,7 +104,7 @@ router.route('/library')
                     owner: playlistOwner,
                     name: name,
                     song: song_id,
-                    translationsChecked: []
+                    translationsChecked: [] //translationsChecked is to store the songs from last export action
```

Comment:
Can you elaborate?
For example:
The user checks off songs the want to export. We store those songs in this array. This is so that if they refresh, they do not lose their selections.